client/src/components/Homepage.jsx

```diff
@@ -1,5 +1,46 @@
+import PetProfile from "./PetProfile";
+
+const OWNER = {
+  firstname: "Cara",
+  lastname: "Coleman",
+  phone: "(555) 555 - 5555",
+  email: "email@email.com",
+  ecfirstname: "Thane",
+  eclastname: "Wilson",
+  ecphone: "(444) 444 - 4444",
+  ecemail: "ecemail@email.com",
+};
+
+const PET = {
+  petname: "Watts",
+  species: "dog",
+  breed: "Airedale Terrier",
+  sex: "male",
+  altered: true,
+  birthday: "2017-05-14",
+  weight: 55,
+  physicaldesc: "Black and tan, curly",
+  staytype: "boarding",
+  checkedin: true,
+};
+
 export default function Homepage() {
-  return (<>
-  <h1>Welcome to your kennel!</h1>
-  </>)
-}
\ No newline at end of file
+  return (
+    <>
+      <h1>Welcome to your kennel!</h1>
+      <PetProfile
```

Comment:
Consider one of these:

1. Just have one attribute, `pet`. I think this is the nicest... much easier to read.
2. Use the spread operator to eliminate all this stuff on the call, ie `{{...PET}}`

Reply:
This is definitely much better than what I've been doing! I've changed this on the current components I'm working on, I'm going to go back and fix this specific one later :)